e2e/libs/volume/crd.py

```diff
@@ -102,6 +103,18 @@ def create(self, volume_name, size, numberOfReplicas, frontend, migratable, data
 
     def delete(self, volume_name, wait):
         try:
+            self.obj_api.patch_namespaced_custom_object(
+                group="longhorn.io",
+                version="v1beta2",
+                namespace=constant.LONGHORN_NAMESPACE,
+                plural="volumes",
+                name=volume_name,
+                body={
+                    "metadata": {
+                        "finalizers": []
+                    }
+                }
+            )
```

Comment:
It seems a workaround, but we need to resolve the issue in the end.
cc @mantissahz